Configuration/TypoScript/ContentElement/Helper/ContentElement.typoscript

```diff
@@ -168,22 +168,28 @@ lib.contentElement {
                 default {
                     breakpoint = 1200
                     width = 1100
+                    widthRetina = 2200
```

Comment:
As mentioned above, we should not configure it by default only enable the option.